README.md

```diff
@@ -125,6 +125,9 @@ The following APIs are shared by Slider and Range.
 | defaultValue | number | `0` | Set initial value of slider. |
 | value | number | - | Set current value of slider. |
 | tabIndex | number | `0` | Set the tabIndex of the slider handle. |
+| aria-label | string | - | Set the `aria-label` attribute on the slider handle.  |
```

Comment:
To avoid confusion, these can be named

`ariaLabelForHandle`, `ariaLabelledByForHandle`, `ariaValueTextFormatterForHandle`